src/utils/APIBreaks/APIBreak.ts

```diff
@@ -180,6 +182,15 @@ export function showAPIBreaks(version: string, { additionalText, apiBreakingChan
           ? `\n\nWe found ${pluralize(detail.totalDetectedLines, "affected line")}.`
           : ""),
     );
+
     ++popUpIndex;
   }
+  //Made terminal more verbose
+  for (const detail of details) {
```

Comment:
I like the idea of showing the info on the terminal, possibly instead of pop-ups. But this is (incorrectly) duplicating logic that the pop-ups use. It should run in the same loop and use the same code.